tests/test__crs.py

```diff
@@ -134,6 +135,29 @@ def test_to_esri_wkt_fix_datum():
     assert 'DATUM["D_North_American_1983"' in _CRS.from_dict(init='epsg:26913').to_wkt(morph_to_esri_dialect=True)
 
 
+@requires_gdal3
+@pytest.mark.parametrize("version", ["WKT2_2019", WktVersion.WKT2_2019])
+def test_to_wkt__version(version):
+    assert _CRS.from_epsg(4326).to_wkt(version=version).startswith('GEOGCRS["WGS 84",DATUM')
```

Comment:
with PROJ 8, this will start instead with GEOGCRS["WGS 84",ENSEMBLE, so:
```suggestion
    assert _CRS.from_epsg(4326).to_wkt(version=version).startswith('GEOGCRS["WGS 84",')
```